client/app/vps/vps.routing.js

```diff
@@ -16,11 +16,18 @@ export default /* @ngInject */($stateProvider) => {
       url: '/{serviceName}',
       redirectTo: 'iaas.vps.detail.dashboard',
       resolve: {
-        stateVps: /* @ngInject */ ($transition$, $q, OvhApiVps) => OvhApiVps.v6().get({
-          serviceName: _.get($transition$.params(), 'serviceName'),
+        capabilities: /* @ngInject */ (
+          serviceName,
+          OvhApiVpsCapabilities,
+        ) => OvhApiVpsCapabilities.Aapi().query({ serviceName }).$promise
+          .then(capabilities => capabilities.map(capability => _.kebabCase(capability))),
+        currentUser: /* @ngInject */ OvhApiMe => OvhApiMe.v6().get().$promise,
```

Comment:
Maybe it could be better to inject `user` at the root state so all child states can benefit from it ?